jenkinsfiles/ginkgo-runtime-kernel.Jenkinsfile

```diff
@@ -28,14 +28,6 @@ pipeline {
                 returnStdout: true,
                 script: 'if [ "${run_with_race_detection}" = "" ]; then echo -n ""; else echo -n "1"; fi'
             )}"""
-        LOCKDEBUG="""${sh(
```

Comment:
I think we want these env vars to be set, as they are passed to runtime VM which causes them to be passed to `make` targets run inside the vm which build the binaries.